### PR TITLE
Feature/filter saved recipes

### PR DESCRIPTION
**Pull request by paulina-isabel**
### What is the feature?
Adding a filter for tags in saved recipes.

### How you implemented the solution?
Added a new dropdown menu in the saved recipes page view.
Created a new function populateSavedTags to load the saved recipe tags in specific dropdown.
populateSavedTags is called in deleteFromSaved() as well as the savedViewBtn event listener.
Added an else statement in the conditional inside viewSavedRecipes to hide the dropdown when the user has no saved recipes available.
When a saved recipe is deleted from saved recipes, its tags get deleted from the dropdown.

### Any changes in the UI (Screenshots)?
ezgif com-video-to-gif (4)

### CODE REVIEW

**Review by alivaditis — APPROVED** [review]
awesome job!!

File: src/data/users.js
Comment (jalbe0076) [inline src/data/users.js:1]: This file needs to stay deleted as we're now fetching API's

**Review by jalbe0076 — CHANGES_REQUESTED** [review]
Minor change, we just need to delete the user data file to meet the comp specs

File: src/data/users.js
Comment (paulina-isabel) [reply src/data/users.js:1]: All done! Good catch.

**Review by CaliHam — APPROVED** [review]
Beautiful work!!

**Review by jalbe0076 — CHANGES_REQUESTED** [review]
please resolve merge conflicts

**Review by jalbe0076 — CHANGES_REQUESTED** [review]
please resolve merge conflicts

**Pull request merged**